src/components/Profile/Interests/styles.scss

```diff
@@ -1,50 +1,48 @@
 .interests-container {
-    position: absolute;
-    bottom: 71px;
-    width: 100%;
-    height: calc(100vh - 490px);
-    overflow-y: auto;
+  position: absolute;
+  bottom: 71px;
+  width: 100%;
+  height: calc(100vh - 490px);
```

Comment:
What is this sorcery? You should not use position absolute. Use position relative and let the components settle where they have space. Actually you don't even need height parameter. Use only padding and margin in sub components.